config/cron-run-all-md2nroff.pl

```diff
@@ -129,6 +129,40 @@ sub verbose {
             if ($out =~ /^\?\?/);
     }
 
+    # Generate a new index.md with all the files that we just
+    # published.  First, read in the header stub.
+    if (!open(IN, "man/index-head.txt")) {
+        chdir("/");
```

Comment:
I need to chdir out of the tmpdir so that the perl destructors can rm -rf the entire tmpdir tree.